code/style.css

```diff
@@ -177,6 +182,23 @@ button {
   margin-bottom: 32px;
 }
 
+.win-or-lose .game-board {
+  margin-left: 27px;
+  margin-bottom: 32px;
+  max-width: 25%;
+  height: auto;
+  align-self: center;
+}
+
+.win-or-lose .card:hover {
+  background-color: var(--light);
+}
+
+.win-or-lose .card:hover img,
+.win-or-lose .card:hover p {
+  display: flex;
+}
+
 @media (max-width: 768px) {
   body {
     flex-direction: column;
```

Comment:
In @media - .game-board, maybe add justify-content and center the images so it looks the same in mobile screen